conferences/2022/javascript.json

```diff
@@ -144,5 +144,16 @@
     "cocUrl": "https://2022.cascadiajs.com/code-of-conduct",
     "offersSignLanguageOrCC": true,
     "url": "https://2022.cascadiajs.com"
+  },
+  {
+    "name": "React Conference Live",
```

Comment:
Apologies, submitted to soon, filling out a new form.

Reply:
No problem. I've just closed it.